tests/test_unit/test_cli/test_behavior.py

```diff
@@ -649,7 +683,7 @@ def test_build_deploy_keep_special_characters(
     my_cdf_toml.cdf.file_encoding = encoding
     monkeypatch.setattr(cdf_toml, "_CDF_TOML", my_cdf_toml)
     BuildCommand(silent=True).execute(
-        False, NAUGHTY_PROJECT, build_dir, None, None, False, env_vars_with_client.get_client(), "raise"
+        False, NAUGHTY_PROJECT, build_dir, ["encoding_issue"], None, False, env_vars_with_client.get_client(), "raise"
```

Comment:
This change is to ensure that this test does not get influenced by other data in the `naughty_project`. I do this by selecting the `encoding_issue` module such that only that is selected.